go/teams/showcase_helper.go

```diff
@@ -111,5 +111,15 @@ func SetTeamMemberShowcase(ctx context.Context, g *libkb.GlobalContext, teamname
 	arg.Args.Add("tid", libkb.S{Val: string(t.ID)})
 	arg.Args.Add("is_showcased", libkb.B{Val: isShowcased})
 	_, err = g.API.Post(arg)
-	return err
+	if err != nil {
+		return err
+	}
+
+	// Clear usercard cache so when user goes to People tab,
+	// fresh card will be loaded.
+	u := g.LoginState().GetUID()
```

Comment:
Can you change this to 

    u := g.ActiveDevice.UID()

Faster, less chance of failing...

Reply:
Should we change other occurrences as well?

Reply:
If there are any others, yes.  Make a ticket?

The other option is `g.Env.GetUID()`, but I prefer the ActiveDevice one as it ensures that it is the uid of a provisioned user.

Reply:
There is just one: https://github.com/keybase/client/blob/master/go/engine/profile_edit.go#L38-L40

this is where I copied this code from. 3 lines didn't seem worth factoring out but I can do that as well if preferable

Reply:
Interesting.  I say go ahead and change that one too.  

I'll make a ticket to delete GetUID() in LoginState...perhaps there is some forgotten reason for it, but I can't think of anything.